Makefile

```diff
@@ -3,7 +3,7 @@ export GOPROXY=https://proxy.golang.org
 
 GO ?= go
 DESTDIR ?=
-EPOCH_TEST_COMMIT ?= $(shell git merge-base HEAD $${DEST_BRANCH:-master})
+EPOCH_TEST_COMMIT ?= $(shell git merge-base $${DEST_BRANCH:-master} HEAD)
```

Comment:
What is the etymology of this variable name? Since it doesn't seem to be used anywhere else, could it be called GIT_MERGE_BASE instead?

Reply:
My best guess is it's used in other `containers/*` repos. and that was copy-pasted over here when it was eventually needed.  Though arguably a bad name, we might consider keeping it to preserve consistency with the other projects.

Reply:
I see it used in buildah also. OK, keep it, I can hold my nose.